lib/ES/Util.pm

```diff
@@ -267,25 +254,47 @@ sub build_single {
     } or do { $output = $@; $died = 1; };
     _check_build_error( $output, $died, $lenient );
 
-    if ( !$lenient ) {
+    unless ( $direct_html ) {
```

Comment:
Everything in here is either moved from above or the same as it was before, just indented one more time.